### PR TITLE
Fix #4909 - Update license to new format

### LINKED ISSUE #4909: Update license to new format
We received permission not to copy the full license in the header of each file.  Instead, we can just have the following:

_OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.
See also https://openstudio.net/license_

Additionally, the wording of the license is SLIGHTLY different. This should replace LICENSE.md and the top of copyright.txt.  

**New wording:**

OpenStudio(R), Copyright (c) 2008, 2023 Alliance for Sustainable Energy, LLC.

Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:

1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.

2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.

3. Redistribution of this software, without modification, must refer to the software by the same designation. Redistribution of a modified version of this software (i) may not refer to the modified version by the same designation, or by any confusingly similar designation, and (ii) must refer to the underlying software originally provided by Alliance as “OpenStudio®”. Except to comply with the foregoing, the term “OpenStudio®”, or any confusingly similar designation may not be used to refer to any modified version of this software or any modified version of the underlying software originally provided by Alliance without the prior written consent of Alliance.

4. The name of the copyright holder(s), any contributors, the United States Government, the United States Department of Energy, or any of their employees may not be used to endorse or promote products derived from this software without specific prior written permission from the respective party.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE UNITED STATES GOVERNMENT, OR THE UNITED STATES DEPARTMENT OF ENERGY, NOR ANY OF THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

### PR DESCRIPTION
**Pull request by jmarrec**
Pull request overview
---------------------

 - Fixes #4909

### Pull Request Author

 - [ ] Model API Changes / Additions
 - [ ] Any new or modified fields have been implemented in the EnergyPlus ForwardTranslator (and ReverseTranslator as appropriate)
 - [ ] Model API methods are tested (in `src/model/test`)
 - [ ] EnergyPlus ForwardTranslator Tests (in `src/energyplus/Test`)
 - [ ] If a new object or method, added a test in NREL/OpenStudio-resources: Add Link
 - [ ] If needed, added VersionTranslation rules for the objects (`src/osversion/VersionTranslator.cpp`)
 - [ ] Verified that C# bindings built fine on Windows, partial classes used as needed, etc.
 - [ ] All new and existing tests passes
 - [ ] If methods have been deprecated, update rest of code to use the new methods

**Labels:**

 - [ ] If change to an IDD file, add the label `IDDChange`
 - [ ] If breaking existing API, add the label `APIChange`
 - [ ] If deemed ready, add label `Pull Request - Ready for CI` so that CI builds your PR

### Review Checklist

This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] Code Style, strip trailing whitespace, etc.
 - [ ] All related changes have been implemented: model changes, model tests, FT changes, FT tests, VersionTranslation, OS App
 - [ ] Labeling is ok
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified

### CODE REVIEW

File: developer/ruby/ApplyCopyright.rb [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+ROOT_DIR = File.absolute_path(File.join(File.dirname(__FILE__), "../../"))
+LICENSE_PATH = File.join(ROOT_DIR, 'LICENSE.md')
+
+
+license_lines = File.readlines(LICENSE_PATH)
+# Now with short version
+license_lines = [
+  "OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.\n",
+  "See also https://openstudio.net/license\n"
+]
```
Comment (jmarrec) [inline developer/ruby/ApplyCopyright.rb:19]: New short license here. Shouldn't change.

the gttps://openstudio.net/license is a dead link currently though

File: developer/ruby/ApplyCopyright.rb [hunk 2]
```diff
@@ -37,25 +37,18 @@
...
+  "empty.hpp"
+]
 
 # glob for hpp and cpp
-files = Dir.glob(basepath + "/src/**/*.[ch]pp")
-files.concat Dir.glob(basepath + "/ruby/**/*.[ch]pp")
-files.concat Dir.glob(basepath + "/src/**/*.cxx.in")
-files.concat Dir.glob(basepath + "/src/**/*.tmp")
+files = Dir.glob(File.join(ROOT_DIR, "**/*.[ch]pp"))
+files += Dir.glob(File.join(ROOT_DIR, "**/*.[ch]xx.in"))
```
Comment (jmarrec) [inline developer/ruby/ApplyCopyright.rb:51]: Clean up the code a bit.
Always

File: LICENSE.md
Comment (jmarrec) [inline LICENSE.md:1]: Modif here

**Comment by tijcolem** [comment]
@jmarrec This is great.  Going to make the copyright maintenance so much easier.

**Review by tijcolem — APPROVED** [review]

**Pull request merged**